apps/web/src/server/applications/case/project-team/applications-project-team.controller.js

```diff
@@ -19,51 +20,67 @@ export async function viewProjectTeamListPage(request, response) {
  *
  * @type {import('@pins/express').RenderHandler<{}, {}, {query: string}, {q: string, number: string}>}
  */
-export async function viewProjectTeamSearchPage({ body, query, errors }, response) {
+export async function viewProjectTeamSearchPage(
+	{ body, query, errors: validationErrors, session },
+	response
+) {
+	const template = `applications/case-project-team/project-team-search`;
 	const searchTerm = body.query?.length ? body.query : query.q;
 	const pageNumber = Number(query.number || '1');
 
-	const templateData = await searchProjectTeamMembersData(searchTerm, pageNumber, errors);
+	// checkpoint 1: render empty search page when query not valid or empty
+	if (validationErrors || !searchTerm) {
+		return response.render(template, { errors: validationErrors });
+	}
+
+	// checkpoint 2: render empty search page when AD token is not found
+	const { errors: authErrors, token } = await projectTeamADService.getTokenOrAuthErrors(session);
+
+	if (authErrors || !token) {
+		return response.render(template, { errors: authErrors });
+	}
 
-	return response.render(`applications/case-project-team/project-team-search.njk`, templateData);
+	// checkpoint 3: search member and render paginated list of results
+	const searchResults = await searchProjectTeamMembersData(searchTerm, token, pageNumber);
+
+	return response.render(template, searchResults);
 }
 
 /**
  * Search project team members and return results
+ *
  * @param {string} searchTerm
+ * @param {string} token
  * @param {number} pageNumber
- * @param {ValidationErrors | undefined} validationErrors
  */
-async function searchProjectTeamMembersData(searchTerm, pageNumber, validationErrors) {
-	let results;
-	let errors = validationErrors;
+async function searchProjectTeamMembersData(searchTerm, token, pageNumber) {
 	let paginationButtons;
 
-	if (!validationErrors && searchTerm) {
-		results = { items: [] };
-		const { errors: apiErrors, results: apiResults } = await searchProjectTeamMembers(
-			searchTerm,
-			pageNumber
-		);
-		errors = apiErrors;
+	/** @type {{items: ProjectTeamMember[]}} */
+	let results = { items: [] };
+
+	const { errors, results: apiResults } = await searchProjectTeamMembers(
+		searchTerm,
+		token,
+		pageNumber
+	);
 
-		if (apiResults) {
-			results = apiResults;
+	if (apiResults) {
+		results = apiResults;
```

Comment:
could use destructuring here

Reply:
no because results is defined as [] before